api/s2n.h

```diff
@@ -612,6 +612,27 @@ extern int s2n_async_pkey_op_apply(struct s2n_async_pkey_op *op, struct s2n_conn
 S2N_API
 extern int s2n_async_pkey_op_free(struct s2n_async_pkey_op *op);
 
+/**
```

Comment:
Given that we're not intending to for this to be public, these declarations should be in the s2n_str.h file.

Reply:
The API is not being released in s2n.h, adding to the utility functions in bin/common.h instead.